game/scripts/npc/abilities/leshrac_diabolic_edict.txt

```diff
@@ -30,7 +30,7 @@
 
     // Damage.
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityDamage"                                       "11 22 33 44 88 110" //OAA
+    "AbilityDamage"                                       "8 20 32 44 68 104"
```

Comment:
Even though this is resolved, I would like to say that he as actually super item based, one of the reasons he is good is his aghs because of the constant slow. Ways to improve him overall would be improving his Q because at early levels it is very inconsistent whereas at later levels it is much easier to land. His edict is only good whilst soloing people because when the damage is spread out, he does nothing, his damage mainly consists of his R plus lightning aghs. The reason his edict is good is because he can chase through his slowing lightning and a stun if he catches them of guard.

Reply:
His aghs is useless, its 1 lightning every 1.75 second(lightning doesnt bounce). His dmg is ult and W. E is mostly just a slow and Q is a stun. Rn W explosion count is picked ant lvl 25 ,not lightning slow durration.